test/models/file_push_test.rb

```diff
@@ -3,7 +3,52 @@
 require "test_helper"
 
 class FilePushTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  setup do
+    @user = users(:luca)
+  end
+  test "should create file push with name" do
+    file_push = FilePush.new(
+      payload: "test payload",
+      name: "Test File Push",
+      user: @user
+    )
```

Comment:
Tests pass, but more suitable attributes can be selected to initialize FilePush. Do you have any suggestion? @pglombardo

Reply:
I updated attributes by adding files and removing payload.